01-messaging-crypto-and-init.md

```diff
@@ -590,21 +578,25 @@ creating a decryption oracle.
 
 The structure of transport messages resembles the following:
 ```
-+------------------------------
-|2-byte encrypted packet length|
-+------------------------------
-| 16-byte MAC of the encrypted |
-|        packet length         |
-+------------------------------
-|                              |
-|                              |
-|          ciphertext          |
-|                              |
-|                              |
-+------------------------------
++----------------------------------
+|2-byte encrypted ciphertext length|
++----------------------------------
+|   16-byte MAC of the encrypted   |
+|        ciphertext length         |
++----------------------------------
+|                                  |
+|                                  |
+|             ciphertext           |
+|                                  |
+|                                  |
++----------------------------------
+|         16-byte MAC of the       |
+|            ciphertext            |
++----------------------------------
 ```
 
-The prefixed packet lengths are encoded as a `2-byte` big-endian integer.
+The prefixed packet lengths are encoded as a `2-byte` big-endian integer,
```

Comment:
That's lightly true: an implementation could simply prepend the twp byte type to the encapsulation of data though.

Certainly pushing it down a layer like this simplifies the alignment problem, which @adiabat was concerned with in Milan IIRC.